GoogleMapsApi/Engine/MapsAPIGenericEngine.cs

```diff
@@ -20,6 +20,8 @@ public abstract class MapsAPIGenericEngine<TRequest, TResponse>
         internal static event UriCreatedDelegate OnUriCreated;
         internal static event RawResponseReciviedDelegate OnRawResponseRecivied;
 
+#if NET45
```

Comment:
hmmm .... Let's think if there's a nicer way?
What about the proposed way here- https://docs.microsoft.com/en-us/dotnet/core/porting/project-structure ?